e2e-tests/portal/cypress/e2e/journeys/maker/edit-deal-name/edit-deal-name.spec.js

```diff
@@ -1,43 +1,34 @@
 const { contract, editDealName, defaults } = require('../../../pages');
-const relative = require('../../../relativeURL');
 const MOCK_USERS = require('../../../../../../e2e-fixtures');
+const { additionalRefName } = require('../../../../fixtures/deal');
 
 const { ADMIN, BANK1_MAKER1 } = MOCK_USERS;
 
 context('Edit deal name', () => {
-  let deal;
-
-  const dummyDeal = {
-    bankInternalRefName: 'abc-1-def',
-    additionalRefName: 'Additional reference name example',
-  };
-
   before(() => {
     cy.deleteDeals(ADMIN);
-    cy.insertOneDeal(dummyDeal, BANK1_MAKER1).then((insertedDeal) => {
-      deal = insertedDeal;
-    });
+
+    cy.createBssEwcsDeal({});
   });
 
   it('rejects an empty field', () => {
-    cy.login(BANK1_MAKER1);
-    contract.visit(deal);
+    cy.loginGoToDealPage(BANK1_MAKER1);
     contract.editDealName().contains('Edit deal name');
     contract.editDealName().click();
 
-    cy.title().should('eq', `Change name - ${deal.additionalRefName}${defaults.pageTitleAppend}`);
-    editDealName.additionalRefName().should('have.value', deal.additionalRefName);
+    cy.title().should('eq', `Change name - ${additionalRefName}${defaults.pageTitleAppend}`);
+    editDealName.additionalRefName().should('have.value', additionalRefName);
     cy.keyboardInput(editDealName.additionalRefName(), '{selectall}{backspace}');
```

Comment:
this is weird (I know its legacy)
```
editDealName.additionalRefName().clear()
```
would be much better

Reply:
can do. @ttbarnes are you happy with the suggested change?

Reply:
Yes much better

Reply:
done as suggested.